main.go

```diff
@@ -89,6 +89,10 @@ func (p Proxy) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyhttp.
 func (p Proxy) authRedirect(w http.ResponseWriter, r *http.Request) (string, error) {
 	token := p.getToken(r)
 
+	q := r.URL.Query()
```

Comment:
Cool, I'm glad to see you found a better way than the hack I used in `Caddyfile`. Speaking of which, please remove this line: `rewrite @static_site {path}?` from the Caddyfile, and edit the comment on the previous line.

Reply:
Looking at those lines, looks like I may need to manually check if this works again. Poor internet has hindered me, but planning on testing this this week.

Reply:
Local testing had no difference here between the two, but will push this one through as it should keep other params in case that is needed.